apps/web/app/landing-page/Hero.tsx

```diff
@@ -40,15 +41,14 @@ export const Hero = () => {
 
   useEffect(() => {
     (async () => {
-      const response = await fetch(ApiRoutes.Subscribers)
+      const response = await fetch(ApiRoutes.Subscribers);
       if (response.ok) {
-        const count = await response.json()
-        setSubscribersCount(count)
```

Comment:
It looks like your linting is adding ";" to the end of the lines, would you please undo these changes and check only what is indeed change?

Thank you in advance!

Reply:
I saw there is PR about eslint/prettier config, it's an issue in this setting. I realized that some files are using `;`

Reply:
https://github.com/ocodista/trampar-de-casa/pull/63#issuecomment-1616167018